MrTimms/MrTimms-Phx.md

```diff
@@ -5,4 +5,5 @@
 - Chino Bandito Takee-Outee
 
 #### Things to do
-
+- Castles N' Coasters
+- Phoenix Zoo
```

Comment:
These sound like lots of fun! I bet it's hot out at the zoo though...are there any good indoor activities? ☀️